src/index.js

```diff
@@ -0,0 +1,175 @@
+import './index.css';
+import reloadDom from './module/clear.js';
+
+const form = document.getElementById('form');
+const listItems = document.getElementById('list-items');
+const clearSelected = document.querySelector('.clear');
+
+class Task {
+  constructor(newData) {
+    this.newData = newData;
+  }
+
+  delete() {
```

Comment:
I think it would be a lot better if you could use an arrow function here to make your codes look more updated :ok_hand: